spec/controllers/state_file/questions/submission_confirmation_controller_spec.rb

```diff
@@ -15,4 +15,33 @@
       expect(response_html).to have_text "state tax return is now submitted"
     end
   end
+
+  describe "nj veteran content" do
+    context "when nj and primary veteran" do
+      let(:intake) { create :state_file_nj_intake, :married_filing_jointly, primary_veteran: "yes", spouse_veteran: "no" }
+      render_views
+      it "shows veteran documentation requirements" do
+        get :edit
+        expect(response_html).to have_text "You said you or your spouse are filing for the veteran exemption"
```

Comment:
[pebble] we typically use the i18n key instead of hard coding the string in tests. makes it easier to change the copy in future